docs/dev/android/setup-android-build-environment.md

```diff
@@ -1,89 +1,75 @@
-   argument required to build OpenSSL against a v21 toolchain).
-    - Extract it, put it somewhere (`$HOME/.android-ndk-r15c` is a reasonable
-      choice, but it doesn't matter), and set `ANDROID_NDK_ROOT` to this location.
-
-2. Install `rustup` from https://rustup.rs:
-    - If you already have it, run `rustup update`
-    - Run `rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android`
-    - Run `rustup toolchain add beta` (TODO: this no longer appears necessary).
-
-3. Ensure your clone of `mozilla/application-services` is up to date.
-
-4. Setup your NDK toolchains.
-    - Create a directory where we'll install the standalone toolchains.
-        - `mkdir -p ~/.ndk-standalone-toolchains`
-        - `export ANDROID_NDK_TOOLCHAIN_DIR=$HOME/.ndk-standalone-toolchains`
-    - `cd path/to/application-services/libs`
-    - `./setup_toolchains_local.sh $ANDROID_NDK_ROOT`
-        - Say yes if/when prompted.
-        - When this is done, it should have set `$ANDROID_NDK_API_VERSION` (to 21),
-          but you should add this to an rcfile as above.
-
-5. Install or locate Java
-    - Either install Java, or, if Android Studio is installed, you can probably find one
-      installed in a `jre` directory under the Android Studio directory.
-    - Set `JAVA_HOME` to this location and add it to your rc file.
-
-6. Install or locate the Android SDKs
-   - Install the Android SDKs. If Android Studio is involved, it may have already installed
-     them somewhere - use the "SDK Manager" to identify this location.
-   - Set `ANDROID_HOME` to this location and add it to your rc file.
-
-7. Build openssl and sqlcipher
-    - `cd path/to/application-services/libs` (Same dir you were just in for step 4)
-    - `./build-all.sh android` (Go make some coffee or something, this will take
-       some time as it has to compile sqlcipher and openssl for x86, x86_64, arm, and arm64).
-    - Note that if something goes wrong here
-        - Check all environment variables mentions above are set and correct.
-        - The following directories should exist, and point to standalone NDK
-          toolchains `$ANDROID_NDK_TOOLCHAIN_DIR/{x86,x86_64,arm,arm64}-$ANDROID_NDK_API_VERSION`.
+- `NDK_HOME`
+
+The easiest way to install all the dependencies (and automatically
+handle updates), is by using [Android Studio](https://developer.android.com/studio/index.html).
+Once this is installed, the following dependencies can be installed through it:
+
+- Android SDK Tools
+- NDK
+- CMake
+- LLDB
+
+To install a dependency, open Android Studio and select `Tools > SDK Manager > SDK Tools`. Then tick the boxes corrensponding to the dependencies listed above.
+
+With the dependencies installed, note down the `Android SDK Location` in `Tools > SDK Manager`. Set the `ANDROID_HOME` environment variable to that path. The `NDK_HOME` can be set to `NDK_HOME=$ANDROID_HOME/ndk-bundle`.
```

Comment:
I didn't need to set `NDK_HOME`. Once I installed the ndk through the sdk manager it picked it up automatically.

Reply:
Oh, I see. This seems required on my desktop, but not required on my laptop :\ Should this stay or should it go? :D

Reply:
If it was needed, keep it in. It certainly won't break stuff.